src/test/fuzz/bloom_filter.cpp

```diff
@@ -25,7 +25,7 @@ void test_one_input(const std::vector<uint8_t>& buffer)
         fuzzed_data_provider.ConsumeIntegral<unsigned int>(),
         static_cast<unsigned char>(fuzzed_data_provider.PickValueInArray({BLOOM_UPDATE_NONE, BLOOM_UPDATE_ALL, BLOOM_UPDATE_P2PUBKEY_ONLY, BLOOM_UPDATE_MASK}))};
     while (fuzzed_data_provider.remaining_bytes() > 0) {
-        switch (fuzzed_data_provider.ConsumeIntegralInRange(0, 4)) {
+        switch (fuzzed_data_provider.ConsumeIntegralInRange(0, 3)) {
```

Comment:
Please keep `fuzzed_data_provider.ConsumeIntegralInRange(0, 4)` in order to not invalidate the existing seed corpus :)

Reply:
I think this is fine. Running the fuzz engine over the existing seeds for a minute or two will adjust the seeds to the new format.

No strong opinion, though.

Reply:
I don't think we should hold back code changes just to not 'invalidate the seed corpus'. This is putting the horse behind the cart.

Reply:
And to go even further: Any code change to master is invalidating some seeds. So with the logic applied that we shouldn't invalidate the existing seeds, we wouldn't be allowed to do any changes to master anymore.

Reply:
The point here is the *only* net effect of from changing `4` to `3` here is that it invalidates the existing seed corpus. Not a big deal of course: treat it as a nit.

The idea that someone would be suggesting that we should hold back otherwise necessary changes not to invalidate the seed corpus is just silly: come on :)

Reply:
Okay, yes, I understand your point now.